// f1-telemetry-client/build/src/mocks/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const PacketCarSetupData_1 = require("./2018/PacketCarSetupData");
exports.PACKET_CAR_SETUP_DATA_BUFFER_2018 = PacketCarSetupData_1.PACKET_CAR_SETUP_DATA_BUFFER_2018;
exports.PACKET_CAR_SETUP_DATA_PARSED_2018 = PacketCarSetupData_1.PACKET_CAR_SETUP_DATA_PARSED_2018;
const PacketCarStatusData_1 = require("./2018/PacketCarStatusData");
exports.PACKET_CAR_STATUS_DATA_BUFFER_2018 = PacketCarStatusData_1.PACKET_CAR_STATUS_DATA_BUFFER_2018;
exports.PACKET_CAR_STATUS_DATA_PARSED_2018 = PacketCarStatusData_1.PACKET_CAR_STATUS_DATA_PARSED_2018;
const PacketCarTelemetryData_1 = require("./2018/PacketCarTelemetryData");
exports.PACKET_CAR_TELEMETRY_DATA_BUFFER_2018 = PacketCarTelemetryData_1.PACKET_CAR_TELEMETRY_DATA_BUFFER_2018;
exports.PACKET_CAR_TELEMETRY_DATA_PARSED_2018 = PacketCarTelemetryData_1.PACKET_CAR_TELEMETRY_DATA_PARSED_2018;
const PacketEventData_1 = require("./2018/PacketEventData");
exports.PACKET_EVENT_DATA_BUFFER_2018 = PacketEventData_1.PACKET_EVENT_DATA_BUFFER_2018;
exports.PACKET_EVENT_DATA_PARSED_2018 = PacketEventData_1.PACKET_EVENT_DATA_PARSED_2018;
const PacketHeader_1 = require("./2018/PacketHeader");
exports.PACKET_HEADER_BUFFER_2018 = PacketHeader_1.PACKET_HEADER_BUFFER_2018;
exports.PACKET_HEADER_PARSED_2018 = PacketHeader_1.PACKET_HEADER_PARSED_2018;
const PacketLapData_1 = require("./2018/PacketLapData");
exports.PACKET_LAP_DATA_BUFFER_2018 = PacketLapData_1.PACKET_LAP_DATA_BUFFER_2018;
exports.PACKET_LAP_DATA_PARSED_2018 = PacketLapData_1.PACKET_LAP_DATA_PARSED_2018;
const PacketMotionData_1 = require("./2018/PacketMotionData");
exports.PACKET_MOTION_DATA_BUFFER_2018 = PacketMotionData_1.PACKET_MOTION_DATA_BUFFER_2018;
exports.PACKET_MOTION_DATA_PARSED_2018 = PacketMotionData_1.PACKET_MOTION_DATA_PARSED_2018;
const PacketParticipantsData_1 = require("./2018/PacketParticipantsData");
exports.PACKET_PARTICIPANTS_DATA_BUFFER_2018 = PacketParticipantsData_1.PACKET_PARTICIPANTS_DATA_BUFFER_2018;
exports.PACKET_PARTICIPANTS_DATA_PARSED_2018 = PacketParticipantsData_1.PACKET_PARTICIPANTS_DATA_PARSED_2018;
const PacketSessionData_1 = require("./2018/PacketSessionData");
exports.PACKET_SESSION_DATA_BUFFER_2018 = PacketSessionData_1.PACKET_SESSION_DATA_BUFFER_2018;
exports.PACKET_SESSION_DATA_PARSED_2018 = PacketSessionData_1.PACKET_SESSION_DATA_PARSED_2018;
const PacketCarSetupData_2 = require("./2019/PacketCarSetupData");
exports.PACKET_CAR_SETUP_DATA_BUFFER_2019 = PacketCarSetupData_2.PACKET_CAR_SETUP_DATA_BUFFER_2019;
exports.PACKET_CAR_SETUP_DATA_PARSED_2019 = PacketCarSetupData_2.PACKET_CAR_SETUP_DATA_PARSED_2019;
const PacketCarStatusData_2 = require("./2019/PacketCarStatusData");
exports.PACKET_CAR_STATUS_DATA_BUFFER_2019 = PacketCarStatusData_2.PACKET_CAR_STATUS_DATA_BUFFER_2019;
exports.PACKET_CAR_STATUS_DATA_PARSED_2019 = PacketCarStatusData_2.PACKET_CAR_STATUS_DATA_PARSED_2019;
const PacketCarTelemetryData_2 = require("./2019/PacketCarTelemetryData");
exports.PACKET_CAR_TELEMETRY_DATA_BUFFER_2019 = PacketCarTelemetryData_2.PACKET_CAR_TELEMETRY_DATA_BUFFER_2019;
exports.PACKET_CAR_TELEMETRY_DATA_PARSED_2019 = PacketCarTelemetryData_2.PACKET_CAR_TELEMETRY_DATA_PARSED_2019;
const PacketEventData_2 = require("./2019/PacketEventData");
exports.PACKET_EVENT_DATA_BUFFER_2019 = PacketEventData_2.PACKET_EVENT_DATA_BUFFER_2019;
exports.PACKET_EVENT_DATA_PARSED_2019 = PacketEventData_2.PACKET_EVENT_DATA_PARSED_2019;
const PacketHeader_2 = require("./2019/PacketHeader");
exports.PACKET_HEADER_BUFFER_2019 = PacketHeader_2.PACKET_HEADER_BUFFER_2019;
exports.PACKET_HEADER_PARSED_2019 = PacketHeader_2.PACKET_HEADER_PARSED_2019;
const PacketLapData_2 = require("./2019/PacketLapData");
exports.PACKET_LAP_DATA_BUFFER_2019 = PacketLapData_2.PACKET_LAP_DATA_BUFFER_2019;
exports.PACKET_LAP_DATA_PARSED_2019 = PacketLapData_2.PACKET_LAP_DATA_PARSED_2019;
const PacketMotionData_2 = require("./2019/PacketMotionData");
exports.PACKET_MOTION_DATA_BUFFER_2019 = PacketMotionData_2.PACKET_MOTION_DATA_BUFFER_2019;
exports.PACKET_MOTION_DATA_PARSED_2019 = PacketMotionData_2.PACKET_MOTION_DATA_PARSED_2019;
const PacketParticipantsData_2 = require("./2019/PacketParticipantsData");
exports.PACKET_PARTICIPANTS_DATA_BUFFER_2019 = PacketParticipantsData_2.PACKET_PARTICIPANTS_DATA_BUFFER_2019;
exports.PACKET_PARTICIPANTS_DATA_PARSED_2019 = PacketParticipantsData_2.PACKET_PARTICIPANTS_DATA_PARSED_2019;
const PacketSessionData_2 = require("./2019/PacketSessionData");
exports.PACKET_SESSION_DATA_BUFFER_2019 = PacketSessionData_2.PACKET_SESSION_DATA_BUFFER_2019;
exports.PACKET_SESSION_DATA_PARSED_2019 = PacketSessionData_2.PACKET_SESSION_DATA_PARSED_2019;
